packages/lesswrong/server/emailComponents/NewPostEmail.tsx

```diff
@@ -57,9 +57,15 @@ const NewPostEmail = ({documentId, reason, hideRecommendations, classes}: {
   const { EmailPostAuthors, EmailContentItemBody, EmailPostDate, EmailFooterRecommendations } = Components;
   if (!document) return null;
   
-  let eventLocation = document.joinEventLink || 'Online event'
-  if (document.isEvent && !document.onlineEvent) {
-    eventLocation = document.location
+  // event location - for online events, attempt to show the meeting link
+  let eventLocation: any = document.location
```

Comment:
I know `any` is not great but I couldn't figure out what type to use for the `<a>` case - my editor suggested `Element`, and I tried a few others, but none of them worked. Let me know if you have suggestions.

Reply:
I think `Element` is a rendered HTML element, whereas this is a react element. In any case, this works for me:
```
let eventLocation: String | JSX.Element = document.location

Reply:
Yup that works, thanks!